rd-net/RdFramework/Impl/SocketWire.cs

```diff
@@ -394,9 +506,11 @@ protected virtual bool AcceptHandshake(Socket socket)
 
     public class Client : Base
     {
-      public Client(Lifetime lifetime, [NotNull] IScheduler scheduler, int port, string optId = null) : this(lifetime, scheduler, new IPEndPoint(IPAddress.Loopback, port), optId) {}
+      public Client(Lifetime lifetime, [NotNull] IScheduler scheduler, int port, string optId = null, TimeSpan? heartBeatInterval = null) : 
```

Comment:
No need to add this parameter into contructor, make is as separate property so one could create Wire with customized interval: new SocketWire.Client(...) {HeartbeatInterval = ...}